### PR TITLE
v2 volume: update documents

### PR DESCRIPTION
**Pull request by derekbit**
Longhorn/longhorn#6021

### CODE REVIEW

File: content/docs/1.6.0/deploy/important-notes/index.md [hunk 1]
```diff
@@ -119,3 +119,11 @@ done
 ```
 
 Once you successfully upgrade to version v1.6.0, you will be able to view information about engine image versions on the UI.
+
+### V2 Volume Supports arm64 Platform
+
+V2 Volume now includes support for the `arm64` platform since Longhorn v1.6.0. Please refer to the [document](../../spdk/prerequisites/) for more information.
```
Comment (jillian-maroket) [inline content/docs/1.6.0/deploy/important-notes/index.md:125]:
```suggestion
### V2 Volume Support for ARM64 Platform

As of Longhorn v1.6.0, volumes using the V2 Data Engine support the ARM64 platform. For more information, see [Prerequisites](../../spdk/prerequisites/).
```

File: content/docs/1.6.0/deploy/important-notes/index.md [hunk 2]
```diff
@@ -119,3 +119,11 @@ done
...
+
+### Longhorn System Upgrade with V2 Volumes
+
+At present, Longhorn does not support live upgrades of v2 volumes. Prior to initiating the Longhorn system upgrade process, ensure to detach all v2 volumes.
```
Comment (jillian-maroket) [inline content/docs/1.6.0/deploy/important-notes/index.md:129]:
```suggestion
### Upgrading Longhorn Systems with V2 Volumes

Longhorn currently does not support live upgrading of V2 volumes. Ensure that all V2 volumes are detached before initiating the upgrade process.
```

File: content/docs/1.6.0/spdk/prerequisites.md [hunk 3]
```diff
@@ -7,10 +7,10 @@ weight: 2
 
 Longhorn nodes must meet the following requirements:
 
-- x86-64 CPU with SSE4.2 instruction support
+- V2 Data Engine supports `amd64` and `arm64` platform
```
Comment (jillian-maroket) [inline content/docs/1.6.0/spdk/prerequisites.md:10]:
```suggestion
- AMD64 or ARM64 CPU
```

File: content/docs/1.6.0/spdk/prerequisites.md [hunk 4]
```diff
@@ -7,10 +7,10 @@ weight: 2
...
   > **NOTICE**
   >
-  > Currently, V2 Data Engine only supports `x86_64` platform.
+  >  `arm64` CPU requires `SSE4.2` instruction support.
```
Comment (jillian-maroket) [inline content/docs/1.6.0/spdk/prerequisites.md:13]:
```suggestion
  >  ARM64 CPUs require SSE4.2 instruction support.
```

File: content/docs/1.6.0/spdk/features.md [hunk 5]
```diff
@@ -3,6 +3,7 @@ title: Features
 weight: 1
 ---
 
+- Support `amd64` and `arm64` platforms
```
Comment (jillian-maroket) [inline content/docs/1.6.0/spdk/features.md:6]:
```suggestion
- Support for AMD64 and ARM64 platforms
```

**Review by jillian-maroket — COMMENTED** [review]
@derekbit Let me know if you have concerns about the changes.

**Review by innobead — APPROVED** [review]

**Pull request merged**